test/coreApi.test.js

```diff
@@ -286,6 +285,149 @@ describe('CoreApi.js',()=> {
+                expect(res.account_status).to.be.equals('PENDING');
+            })
+    })
+
+    // expected the API call will fail because of not click activation from linkPaymentAccount
+    it('able to unlink account',async () => {
+        let core = new CoreApi(generateConfig());
+        let parameter = generateParamGopay();
+        let response = await core.linkPaymentAccount(parameter)
+        let accountId = response.account_id;
+        return core.unlinkPaymentAccount(accountId)
+            .catch((e)=>{
+                expect(e.message).to.includes('412');
+                expect(e.httpStatusCode).to.equals('412');
+            })
+    })
+
+    it('fail to get pay account with dummy account',()=>{
+        let core = new CoreApi(generateConfig());
+        return core.getPaymentAccount("dummy")
+            .catch((e)=>{
+                expect(e.message).to.includes('404');
+                expect(e.httpStatusCode).to.equals('404');
+            })
+    })
+
+    it('fail to unlink pay account with dummy account',()=>{
+        let core = new CoreApi(generateConfig());
+        return core.unlinkPaymentAccount("dummy")
+            .catch((e)=>{
+                expect(e.message).to.includes('404');
+                expect(e.httpStatusCode).to.equals('404');
+            })
+    })
+
+    it('able to create subscription',()=>{
+        let core = new CoreApi(generateConfig());
+        let parameter = generateParamSubscription();
+        return core.createSubscription(parameter)
+            .then((res)=>{
+                expect(res.status).to.be.equals('active');
+            })
+    })
+
+    it('able to get subscription',async () => {
+        let core = new CoreApi(generateConfig());
+        let parameter = generateParamSubscription();
+        let response = await core.createSubscription(parameter)
+        let subscriptionId = response.id;
+        return core.getSubscription(subscriptionId)
+            .then((res) => {
+                expect(res.status).to.be.equals('active');
+            })
+    })
+
+    it('able to disable subscription',async () => {
+        let core = new CoreApi(generateConfig());
+        let parameter = generateParamSubscription();
+        let response = await core.createSubscription(parameter)
+        let subscriptionId = response.id;
+        return core.disableSubscription(subscriptionId)
+            .then((res) => {
+                expect(res.status_message).to.be.equals('Subscription is updated.');
+            })
+    })
+
+    it('able to enable subscription',async () => {
+        let core = new CoreApi(generateConfig());
+        let parameter = generateParamSubscription();
+        let response = await core.createSubscription(parameter)
+        let subscriptionId = response.id;
+        return core.enableSubscription(subscriptionId)
+            .then((res) => {
+                expect(res.status_message).to.be.equals('Subscription is updated.');
+            })
+    })
```

Comment:
what's the final state of the subscription after this test file ran? is it `enable`? better make it disabled to prevent unnecessary active subscription on sandbox.
Ref:
https://github.com/Midtrans/midtrans-python-client/blob/master/tests/test_subscription.py#L81-L90